docusaurus/docs/create-a-plugin/extend-a-plugin/add-anonymous-usage-reporting.md

```diff
@@ -123,6 +124,7 @@ export function QueryEditor(props: EditorProps): ReactElement {
       rawQuery: type,
     });
 
+    //  fetch this report with two arguments
```

Comment:
Better something like `// send the metric using the report function` 

You are technically sending a report, not fetching a report with this function

Reply:
done.